src/Knp/Menu/Renderer/ListRenderer.php

```diff
@@ -130,7 +131,11 @@ protected function renderItem(ItemInterface $item, array $options): string
         if ($this->matcher->isCurrent($item)) {
             $class[] = $options['currentClass'];
         } elseif ($this->matcher->isAncestor($item, $options['matchingDepth'])) {
-            $class[] = $options['ancestorClass'];
+            if(isset($options['ancestorClass']) { // Deprecated: it will be removed in future (@see: ancestor_class)
```

Comment:
this "if" should be spaced. Did you run php-cs-fixer?

Reply:
Done in https://github.com/KnpLabs/KnpMenu/pull/351/commits/d2b325b7d33a185bde3afa918bd1403ce6aab0c2

I didn't see any .php-cs-fixer.php config in this repo, so I don't know the exactly config requested. Maybe it would be useful to add it, and also some static code analyzer software after push commits like php-cs-fixer project does in other PR.

Reply:
The config was added a few minutes ago